docs/codingpractises.md

```diff
@@ -0,0 +1,19 @@
+# Coding Practises
+
+The [contributing.md](Contributing Guide) provides (among other things) guidance on the workflows and general expectations associated with contributing a code change to `scores`. This document eschews some of the context to focus on specifying technical information needed when developing code for `scores`.
+
+# Expectations:
+
+1. Black has been run over the code
+2. Pylint has been run and there are zero lint issues
+3. mypy has been run and there are zero mypy issues
```

Comment:
Inconsistent use of back ticks for package names since you used it for `scores` in the previous paragraph.

```suggestion
1. `black` has been run over the code
2. `pylint` has been run and there are zero lint issues
3. `mypy` has been run and there are zero type issues
```

Maybe adding something along the lines of "for target versions of linters and fixters refer to the [dev dependencies in pyproject.toml](Link to file)" to this guide would be good. I expect this could come a point of contention later on if we ask devs to repeat linter steps when their local versions conflict with our pinned ones.